File: engineering/codebase/smartqr.frontend-services/src/domain/codes/content/enums/MobileAppStoreType.ts
/** Defines which app store a mobile-app link points at. */
export const MobileAppStoreType = {
  /** Apple App Store. */
  AppStore: "appStore",

  /** Google Play. */
  PlayStore: "playStore",

  /** Any other store or a direct download page. */
  Other: "other",
} as const;

export type MobileAppStoreType = (typeof MobileAppStoreType)[keyof typeof MobileAppStoreType];

File: engineering/codebase/smartqr.backend-services/SmartQr.Common.Domain/Codes/Content/MobileApp/Enums/MobileAppStoreType.cs
namespace SmartQr.Common.Domain.Codes.Content.MobileApp.Enums;

/// <summary>Defines which app store a mobile-app link points at.</summary>
public enum MobileAppStoreType
{
    /// <summary>Represents the Apple App Store.</summary>
    AppStore,

    /// <summary>Represents the Google Play store.</summary>
    PlayStore,

    /// <summary>Represents any other store or a direct download page.</summary>
    Other,
}
